src/front-end/src/App.css

```diff
@@ -28,12 +36,43 @@ a {
   margin: 30px;
 }
 
+.MentorSuggested {
+  border: solid black;
+  padding: 25px;
+  margin: 30px;
+}
+
+.MentorAccepted {
+  border: solid black;
+  padding: 25px;
+  margin: 30px;
+}
+
+.MentorTutorial {
+  border: solid black;
+  padding: 25px;
+  margin: 30px;
+}
+
+#logout_button {
+  position:absolute;
+  right:0;
+}
+
 .MentorForm {
   border: solid black;
   padding: 25px;
   margin: 30px;
 }
```

Comment:
We should probably add some variance to the UI. In the future, to reduce redundancy (since the styling is all the same between the className attributes for the Froms and Login page), they can be grouped together like this:
.MenteeForm,
.MentorForm,
.Login  {
  border: solid black;
  padding: 25px;
  margin: 30px;
}
I would maybe remove the border from the Tutorial, Accepted, and Suggested mentee tabs so that the interface has a slightly different look to it.

Reply:
There is only a border on the tutorial page to make the text look nice and similar to the rest which I think looks good, but if you think I should change it to without the border that is okay too (whichever way the group agrees is totally fine with me)! And thank you for telling me about the grouped Elements i didnt know about that